### DISCUSSION TITLE
[BUG] Taskbar for primary monitor is unusally wide when on left or right

### DISCUSSION THREAD

**Discussion by qiushanbeikun** [valinet/ExplorerPatcher#1474] [General]
![image](https://github.com/valinet/ExplorerPatcher/assets/33504449/5207144c-740d-4837-aa71-06d8cc32eff1)
As in the picture, LTS and LTS windows 11
This only happens to the primary monitor.

**Comment by apmReaper** [comment]
Bumping this as I'm having the exact same problem and would also love to know the solution, since re-installing didn't work.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2964

please close this.
